recipes/pyamg/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+    name: pyamg
+    version: "3.0.2"
+
+source:
+    git_url: https://github.com/pyamg/pyamg.git
+    git_tag: v3.0.2
+
+build:
+    number: 2
+    string: np{{CONDA_NPY}}py{{CONDA_PY}}_{{PKG_BUILDNUM}}
+
+requirements:
+    build:
+        - python >=2.7,<3|>=3.4,{{PY_VER}}*
```

Comment:
Just leave `python` and let `conda-build-all` and `conda-smithy` do the rest for you :wink: